logic-building/secondSmallestNumber.js

```diff
@@ -0,0 +1,18 @@
+//Given an array of strings, write a function to sort the strings based on their length, 
```

Comment:
is this the correct question of the given answer?

Reply:
changed question